### ISSUE TITLE
Issues adapting the recordings paths

### ISSUE THREAD

**Issue by xeruf** [manjaro-sway/manjaro-sway#870]
### branch

stable

### Architecture

x86_64

### Description

Just discovered the recordings features on accident and did a quick test recording, but it does not appear in the advertised location: 
![image](https://github.com/user-attachments/assets/c06b0361-3169-4ca1-b9d2-631a7e07be7e)

### Reproduction steps

```bash
- start recording
- check recording folder
```

**Comment by boredland** [comment]
You sure? 
[recording_20241110-192031.webm](https://github.com/user-attachments/assets/d3b807f1-f54e-4707-b037-71ea33d6e915)

**Comment by xeruf** [comment]
Of course, my XDG dirs are adjusted ;) but the path it chose is very much correct
https://forge.ftt.gmbh/janek/dotfiles/src/branch/main/.config/user-dirs.dirs%23%23user.janek
